packages/core/src/components/slider/slider.tsx

```diff
@@ -72,12 +72,18 @@ export class Slider extends CoreSlider<ISliderProps> {
         );
     }
 
-    protected handleTrackClick(event: React.MouseEvent<HTMLElement>) {
+    protected handleTrackClick(event: MouseEvent | React.MouseEvent<HTMLElement>) {
```

Comment:
I've checked code, handleTrackClick/Touch is protected and will newer be called with native events, it's safe to remove it.